dftool

```diff
@@ -462,8 +462,16 @@ Transactions are signed with envvar 'DFTOOL_KEY`.
     token_symbol = B.Simpletoken.at(TOKEN_ADDR).symbol().upper()
     token_symbol = token_symbol.replace("MOCEAN", "OCEAN")
     rewards = csvs.loadRewardsCsv(CSV_DIR, token_symbol)
+
+    # "flaten" the rewards dict to dispense all chains in one go
+    all_rewards = {}
+    for chainID in rewards:
+        for lp in rewards[chainID]:
+            all_rewards[lp] = rewards[chainID][lp]
+
```

Comment:
If we wanted, we could also simplify [calcrewards.py](https://github.com/oceanprotocol/df-py/blob/main/util/calcrewards.py), and the shape of the rewards csv, to not have chainID. 

However, that's a fair bit of work. And, there's a chance that we may want to distribute to >1 chain, for blockchain partners. Therefore, I am _not_ recommending it for here.